app/src/main/java/com/hannesdorfmann/adapterdelegates3/sample/DiffActivity.java

```diff
@@ -0,0 +1,143 @@
+package com.hannesdorfmann.adapterdelegates3.sample;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Toast;
+
+import com.hannesdorfmann.adapterdelegates.sample.R;
+import com.hannesdorfmann.adapterdelegates3.DiffItem;
+import com.hannesdorfmann.adapterdelegates3.sample.model.DiffCat;
+import com.hannesdorfmann.adapterdelegates3.sample.model.DiffDog;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by six_hundreds on 15.06.18.
+ */
```

Comment:
Please remove this comment or make it more meaningful (I think removing it is better).

Reply:
Sure